configs/app/features/getGasButton.ts

```diff
@@ -0,0 +1,36 @@
+import type { Feature } from './types';
+import type { GasRefuelProviderConfig } from 'types/client/gasRefuelProviderConfig';
+
+import { getEnvValue, parseEnvJson } from '../utils';
+import marketplace from './marketplace';
+
+const value = parseEnvJson<GasRefuelProviderConfig>(getEnvValue('NEXT_PUBLIC_GAS_REFUEL_PROVIDER_CONFIG'));
+
+const title = 'Get gas button';
+
+const config: Feature<{
+  name: string;
+  logoUrl?: string;
+  urlTemplate: string;
+  dappId?: string;
+  usdThreshold: number;
+}> = (() => {
+  if (value) {
+    return Object.freeze({
+      title,
+      isEnabled: true,
+      name: value.name,
+      logoUrl: value.logo,
+      urlTemplate: value.url_template,
```

Comment:
chain Id is required, you can make an URL right here once, not to do it inside the component on each render